apps/test-suite/stories/1-components/BlurView/BlurViewScreen.js

```diff
@@ -0,0 +1,101 @@
+import { BlurView } from 'expo';
+import React from 'react';
+import { View, Image, StyleSheet, Text } from 'react-native';
+
+import UIExplorer, {
+  AppText,
+  Code,
+  Description,
+  DocItem,
+  Section,
+  storiesOf,
+} from '../../ui-explorer';
+
+const image = {
+  uri:
+    'https://www.tesla.com/tesla_theme/assets/img/_vehicle_redesign/roadster_and_semi/roadster/hero-portrait@2.jpg',
```

Comment:
download this image to the repo so we don't depend on tesla.com

Reply:
oh yeah some bundler issue